internal/config/init.go

```diff
@@ -71,7 +71,7 @@ func Initialise(
 		return nil, fmt.Errorf("invalid hash algorithm: %s", InitFlag.ServiceKeyHashAlgo)
 	}
 
-	state, err := flowkit.Init(readerWriter, sigAlgo, hashAlgo)
+	state, err := flowkit.Init(readerWriter)
```

Comment:
I think this is a case where we should have the emulator account

Reply:
Good catch. Added